simulator.py

```diff
@@ -596,8 +536,107 @@ def _select_location(self, location_type, city):
         loc = self.rng.choice(cands, p=_normalize_scores(scores))
         visited_locs[loc] += 1
         return loc
+ ################################## PRIVACY & INFERENCE ########################################
```

Comment:
@mweiss17  I meant this form of segregation. It is not professional, but still something to manage a large file with different moving parts in it.